### PR TITLE
fix linter

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!tastemetitus

File: manifest.json [hunk 1]
```diff
@@ -6,15 +6,15 @@
         "en": "Manage PostgreSQL databases over the web",
         "fr": "Application web de gestion des bases de données PostgreSQL"
     },
-    "version": "4-5.7~ynh1",
+    "version": "5.7~ynh1",
```
Comment (Josue-T) [inline manifest.json:9]: Well, the real version is 4-5.7 not 5.7 so we can't just remove this. The main version of pgadmin is "pgadmin 4"

**Review by Josue-T — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 1 above)
Comment (Josue-T) [reply manifest.json:9]: I made a issue about this problem here: https://github.com/YunoHost/package_linter/issues/102

Comment (ericgaspar) [reply manifest.json:9]: fair enough.

**Pull request closed without merging**